umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10/spatial.clj

```diff
@@ -115,42 +115,49 @@
+                                                      :LocalCoordinateSystem local-coordinate-sys
+                                                      :HorizontalDataResolution horizontal-data-resolutions}}
+                     {:ResolutionAndCoordinateSystem nil})]
     (util/remove-nil-keys
+     (merge
       {:Geometry (parse-geometry geom)
-       :ZoneIdentifier (value-of spatial-coverage "Zone_Identifier")
-       :ResolutionAndCoordinateSystem {:GeodeticModel geodetic-model
-                                       :LocalCoordinateSystem local-coordinate-sys
-                                       :HorizontalDataResolution horizontal-data-resolutions}})))
+       :ZoneIdentifier (value-of spatial-coverage "Zone_Identifier")}
+       resolution))))
 
 (defn parse-spatial
   "Returns UMM-C spatial map from DIF 10 XML document."
   [doc]
-  (let [[spatial] (select doc "/DIF/Spatial_Coverage")]
-    {:SpatialCoverageType (dif10-spatial-type->umm-spatial-type (value-of spatial "Spatial_Coverage_Type"))
-     :GranuleSpatialRepresentation (value-of spatial "Granule_Spatial_Representation")
-     :HorizontalSpatialDomain      (parse-horizontal-spatial-domains doc)
-     :VerticalSpatialDomains       (spatial-conversion/convert-vertical-spatial-domains-from-xml
-                                    (select spatial "Vertical_Spatial_Info"))
-     :OrbitParameters             (let [[o] (select spatial "Orbit_Parameters")]
-                                     (as->{:SwathWidth (value-of o "Swath_Width")
-                                           :OrbitPeriod (value-of o "Period")
-                                           :InclinationAngle (value-of o "Inclination_Angle")
-                                           :NumberOfOrbits (value-of o "Number_Of_Orbits")
-                                           :StartCircularLatitude (value-of o "Start_Circular_Latitude")} op
-                                           ;; Add assumed units for the corresponding fields.
-                                           (if (:SwathWidth op)
-                                             (assoc op :SwathWidthUnit "Kilometer")
-                                             op)
-                                           (if (:OrbitPeriod op)
-                                             (assoc op :OrbitPeriodUnit "Decimal Minute")
-                                             op)
-                                           (if (:InclinationAngle op)
-                                             (assoc op :InclinationAngleUnit "Degree")
-                                             op)
-                                           (if (:StartCircularLatitude op)
-                                             (assoc op :StartCircularLatitudeUnit "Degree")
-                                             op)))}))
+  (let [[spatial] (select doc "/DIF/Spatial_Coverage")
+        [o] (select spatial "Orbit_Parameters")]
```

Comment:
I get yelled at when I do this in python by the linter, can you maybe give a longer name?

Reply:
I changed the name to orbit-parameters